backend/python/app/middlewares/auth.py

```diff
@@ -198,7 +198,10 @@ def wrapper(root, info, *args, **kwargs):
                         story_translation_id=kwargs["story_translation_id"],
                     )
                 )
-            else:
+            elif (
+                "story_translation_content_id" in kwargs
+                or "story_translation_contents" in kwargs
+            ):
```

Comment:
Not sure if this should be `elif` or `else`. I could also split it into two `elif`s, and remove the intermediate `story_translation_content_id` variable.